src/rt_core_v2/metadata.py

```diff
@@ -0,0 +1,123 @@
+from enum import Enum
+
+"""Enum for when the string representation of an enum instance is the value"""
+
+
+class ValueEnum(Enum):
+    def __str__(self):
+        return str(self.value)
+
+
+class TupleEventType(ValueEnum):
+    """Each metadata or D tuple represents an action taken on another tuple, which can include
+    (1) insertion of a tuple, (2) marking a tuple as being in error, or
+    (3) retracting an assertion of being in error"""
+
+    INSERT = 1
+    INVALIDATE = 2
+    REVALIDATE = 3
+
+
+class RtChangeReason(ValueEnum):
+    """For each event that a D tuple represents, what is the reason we are doing it?
+    These reasons include:
+      (1) change in belief
+      (2) change in reality
+      (3) change in relevance
+      (4) recognition of an error or mistake"""
+
+    BELIEF = 4
+    REALITY = 5
+    RELEVANCE = 6
+    MISTAKE = 7
```

Comment:
We discussed removing MISTAKE, perhaps it's in a future PR

Reply:
It is in a future PR.